.github/workflows/compileLint.yml

```diff
@@ -1,18 +1,24 @@
-name: Lint
+name: Compile and lint
```

Comment:
This should probably be called Build instead. `build` lints and compiles

Reply:
It isn’t building anything because it produces no output, it simply checks if stuff works

Reply:
We should think of a better name than this though